### DISCUSSION TITLE
Migrating Strongswan IKEv2 setup to a different machine

### DISCUSSION THREAD

**Discussion by gspannu** [strongswan/strongswan#847] [Q&A] [answered]
I have setup _StrongSwan_ VPN server and have a lot of clients connecting to the server
_(Debian server hosted in the cloud)_.

- All settings are based on a Domain name, no IP addresses involved.
- I am not using any additional plugins.
- I am only using iKEv2
- All my settings are effectively in a single file (ipsec.conf)

- I have also created iOS and macOS mobileconfig files that contain the settings for each client.
- Essentially, I have a system that is well configured and provides a VPN service for end users.

**I now need to migrate my setup to a different machine.**

Question: What files/ folders would I need to transfer from the existing machine to the new one, so that all my certs, settings, etc... work on the new machine as well?

I understand that I will have to make changes to iptables, firewall rules, etc... and I already have scripts for those.
All I need is some assistance in knowing the exact files/ folders I need to copy across after I have installed strongman on the new machine.
apt install strongswan strongswan-pki

Thanks...

**Comment by tobiasbrunner** [comment, marked as answer]
I guess you don't actually mean strongMan as that's a completely different project. For strongSwan you just need to copy the config files/directories you modified (see [ConfigurationFiles](https://wiki.strongswan.org/projects/strongswan/wiki/ConfigurationFiles)).

**Reply by gspannu** [reply]
Yes, I meant StrongSwan (auto correction working overtime !) 🤯

Thanks for your response 👍 
I will build a script for the same and test...
